### PR TITLE
New 2.22.4 release post

### PR DESCRIPTION
**Pull request by petersmythe**

### CODE REVIEW

File: _posts/2023-06-19-geoserver-2-22-4-released.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+This release addresses security vulnerabilities and is considered an essential update for production systems.
+
+* [GEOS-10949](https://osgeo-org.atlassian.net/browse/GEOS-10949) Control remote resources accessed by GeoServer
+
+See project [security policy](https://github.com/geoserver/geoserver/blob/main/SECURITY.md) for more information on how security vulnerabilities are managed. 
+
+### New Security > URL Checks page
+
+This release adds a new Check URL facility under the Security menu.  This allows administrators to mitigate the Service Side Request Forgery (SSRF) vulnerabilities implicit in common OGC standards.
```
Comment (mprins) [inline _posts/2023-06-19-geoserver-2-22-4-released.md:40]:
```suggestion
This release adds a new Check URL facility under the Security menu. This allows administrators to mitigate the Server-Side Request Forgery (SSRF) vulnerabilities implicit in common OGC standards.

```

File: _posts/2023-06-19-geoserver-2-22-4-released.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+GeoServer [2.22.4](/release/2.22.4/) release is now available
+with downloads (
+[bin](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22.4/geoserver-2.22.4-bin.zip/download),
+[war](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22.4/geoserver-2.22.4-war.zip/download),
+[windows](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22.4/GeoServer-2.22.4-winsetup.exe/download))
+, along with 
+[docs](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22.4/geoserver-2.22.4-htmldoc.zip/download) and
+[extensions](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22.4/extensions/).
+
+This is a maintenance release of GeoServer providing a existing installations with minor updates and bug fixes.
```
Comment (jodygarnett) [inline _posts/2023-06-19-geoserver-2-22-4-released.md:25]:
```suggestion
This is a maintenance release of GeoServer providing existing installations with minor updates and bug fixes.

```

File: _posts/2023-06-19-geoserver-2-22-4-released.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+GeoServer is made in conjunction with GeoTools 28.4.
```
Comment (jodygarnett) [inline _posts/2023-06-19-geoserver-2-22-4-released.md:26]:
```suggestion
GeoServer 2.24.4 is made in conjunction with GeoTools 28.4. 

```

**Review by mprins — COMMENTED** [review]
this PR seems to duplicate part of #154 (or voce-versa)

File: _posts/2023-06-19-geoserver-2-22-4-released.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+[war](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22.4/geoserver-2.22.4-war.zip/download),
+[windows](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22.4/GeoServer-2.22.4-winsetup.exe/download))
+, along with 
+[docs](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22.4/geoserver-2.22.4-htmldoc.zip/download) and
+[extensions](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22.4/extensions/).
+
+This is a maintenance release of GeoServer providing a existing installations with minor updates and bug fixes.
+GeoServer is made in conjunction with GeoTools 28.4. 
+
+Thanks to Jody Garnett (GeoCat) and Peter Smythe (AfriGIS) for making this release.
```
Comment (jodygarnett) [inline _posts/2023-06-19-geoserver-2-22-4-released.md:28]:
```suggestion
Thanks to Peter Smythe (AfriGIS) and Jody Garnett (GeoCat) for making this release. This is Peter's first time making a GeoServer release and we would like to thank him for volunteering.

```

**Review by jodygarnett — COMMENTED** [review]
Feedback!

**Comment by jodygarnett** [comment]
> this PR seems to duplicate part of #154 (or vice-versa)

Yes, @petersmythe was getting a bit stuck trying to update the README instructions at the same time as making these PRs. 

The README changes are non functional so I am not going to worry about it

File: _posts/2023-06-19-geoserver-2-22-4-released.md (hunk 3 above)
Comment (petersmythe) [reply _posts/2023-06-19-geoserver-2-22-4-released.md]: @jodygarnett This should be GeoServer 2.22.4

**Comment by petersmythe** [comment]
@jodygarnett the Security section seems to have disappeared, along with the screenshot

### New Security > URL Checks page

**Pull request merged**